pkgs/tools/filesystems/dbxfs/default.nix

```diff
@@ -0,0 +1,101 @@
+{ lib, fetchgit, python3Packages, fusePackages }:
+
+let
+  block-tracing = python3Packages.buildPythonPackage rec {
+    pname = "block_tracing";
+    version = "1.0.1";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "9faa009a702a8f2f605ebb78314d5ca2a2a93543d061038a3d3a978c93385e68";
+    };
+  };
+  privy = python3Packages.buildPythonPackage rec {
+    pname = "privy";
+    version = "6.0.0";
+    format = "wheel";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      format = "wheel";
+      sha256 = "e68679bb4006ce83206d9a7af1158cf4d56a2ed6861ee39276907be618dfb8d9";
+    };
+
+    propagatedBuildInputs = [
+      python3Packages.cryptography
+      python3Packages.argon2_cffi
+    ];
+  };
+  fusepyng = python3Packages.buildPythonPackage rec {
+    pname = "fusepyng";
+    version = "1.0.7";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "sha256-i09u+B6GAPI9p1CRaayyYVWC7xFtRqKhrUt+Uw2PiZ8=";
+    };
+
+  };
+  userspacefs = python3Packages.buildPythonPackage rec {
+    pname = "userspacefs";
+    version = "2.0.5";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "sha256-XW9f+m05SI8cdDfF6q6Pq/SRbKzqtIOjFzoY5nibGGw=";
+    };
+
+    buildInputs = [
+      fusePackages.fuse_2
+    ];
+
+    # crashes looking for libfuse in build_ext (?)
+    doCheck = false;
+
+    propagatedBuildInputs = [
+      fusepyng
+    ];
+  };
```

Comment:
Please add those to pythonPackages to instead.

Reply:
I considered that but wouldn't that mean I'd be the maintainer for those packages? I'm only interested in maintaining this dbxfs application really.

Reply:
> I considered that but wouldn't that mean I'd be the maintainer for those packages?

You are the maintainer of those packages anyway 🙃 but if they sit in this file they are likely to be out of date and not receive important updates and most python updates are usually done by python-unstable runs anyway.

> I'm only interested in maintaining this dbxfs application really.

Well, if dependencies are missing there is no real choice. Also adding things like pythonImportChecks or proper checks make sure the packages don't silently break.